jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,817 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+Nim version 1.6 is now officially released!
+
+A year in the making, 1.6 is the latest stable release and by far the largest yet.
+We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone.
+Here are some stats:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in >40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and last 2 months it entered the top-50 (https://forum.nim-lang.org/t/8297).
+We hope this release will reinforce this trend, building on Nim's core strenghs:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; LISP-like flexibility; strong C, C++, JS, python interop;
+and best-in class metaprogramming.
+
+This release includes improvements in the following areas:
+* new language features (user defined literals, private imports, strict effects, `iterable[T]`, new style concepts, dot-like operators, block arguments with optional params)
+* new compiler features (`nim --eval:cmd`, custom nimscript extensions, customizable compiler messages)
+* major improvements to `--gc:arc`, `--gc:orc`
+* correctness and performance of integer and float parsing and rendering in all backends
+* significant improvements in error messages, showing useful context
+* doc generation logic and documentation, in particular `runnableExamples` now works in more contexts and replaces `code-block`.
```

Comment:
This implies `code-block` is deprated, is that the case?

Reply:
if we can merge this PR https://github.com/nim-lang/Nim/pull/18604, then I could change the statement to "now works in every context" instead of "more", and then code-block for nim code can indeed be considered deprecated within nim files (but could have occasional use for non-nim code within nim files, as well as its non-deprecated use in rst files)

Reply:
> This implies code-block is deprated, is that the case?

It is not the case.

Reply:
we might want to reword this then: just remove "and replaces `code-block`."